profiles/fscloud/variables.tf

```diff
@@ -252,6 +259,16 @@ variable "load_balancers" {
     )) == 0
   }
 
+  validation {
+    error_message = "For Network Load Balancer this attribute is required and should be set to `network-fixed`. For Application Load Balancer it is not required."
+    condition = length(
+      flatten([
+        for load_balancer in var.load_balancers :
+        true if !contains(["network-fixed", null], load_balancer.profile)
+      ])
+    ) == 0
+  }
```

Comment:
I don't think we should be duplicating validation for any variable in the fscloud submodule. It will use the validation from the root level module right? So instead of maintaining it in two place (and having the potential for them to become out of sync), can we just remove it? (just double check this works)

Reply:
The only place a validation would be needed is when there are FSCloud-specific validations.

Having said that currently root variables and fscloud variables have almost identical validations, and I can remove all of them. The reason why we don't need them is that  FSCloud calls the root module for the execution.

Reply:
yep thats what I was thinking. Just double check that is the case though